test/test_afba.jl

```diff
@@ -24,28 +24,27 @@ x_star = [-3.877278911564627e-01, 0, 0, 2.174149659863943e-02, 6.168435374149660
 stuff = [
   Dict( "theta"      => 2,
         "mu"         => 0,
-        "it"         => (47,125,93),
+        "it"         => (70,135,100),
```

Comment:
These values needed to be changed because tests were otherwise failing? (In any case I agree with turning them into upper bounds)